### PR TITLE
Add PhoneType in mme-exporter

### PR DESCRIPTION
**Pull request by woojoong88**

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

File: pyserver/conf/mme_exporter.json [hunk 1]
```diff
@@ -10,5 +10,15 @@
         "edgeName": "edge-onf-menlo",
         "tac": 17
       }
+    ],
+    "phoneTypes": [
```
Comment (thakurajayL) [inline pyserver/conf/mme_exporter.json:14]: Hmmm... :) Its something which actually DPI should do. Worried about sim swapping. Was wondering if we could do based on IMEI ?

Comment (woojoong88) [reply pyserver/conf/mme_exporter.json:14]: If we can map IMSI-IMEI and MME can send IMEI to the Flask server, then we can use IMEI for this.

Comment (woojoong88) [reply pyserver/conf/mme_exporter.json:14]: @badhrinathpa Any idea?

Comment (thakurajayL) [reply pyserver/conf/mme_exporter.json:14]: Doable but now now. I will open a Jira ticket with spec reference. We need some changes on mme to get the IMEI from device through signalling.

**Review by badhrinathpa — APPROVED** [review]
+1

**Pull request merged**